src/modulesFromList.nix

```diff
@@ -23,6 +26,11 @@ let
 
     let
 
+      # To allow for the default to just pass a list
+      # or pass an attrset with `paths` and `_import`
+      paths = args.paths or args;
+      _import = args._import or import;
```

Comment:
I did not know the `or` syntax works like this. If `args` is a list the `or` fallback still works.